### PR TITLE
Translate:  Translate the aggregator-microservices readme to Spanish [Issue #2335 (Task of issue #2277 )]

### PR DESCRIPTION
**Pull request by KarmaTashiCat**
Translated [aggregator-microservices/README.md](https://github.com/iluwatar/java-design-patterns/blob/master/aggregator-microservices/README.md) and saved translation to [/localization/es/](https://github.com/iluwatar/java-design-patterns/tree/master/localization/es) ./aggregator-microservices/README.md 

Note: Sorry for first commit, it was a mistake of unsaved version of second(final) commit. After second commit the translation is updated with all changes.

### CODE REVIEW

**Comment by KarmaTashiCat** [comment]
Translate the aggregator-microservices readme to Spanish [Issue #2335 (Task of issue #2277 )]

File: localization/es/aggregator-microservices/README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+---
+título: Agregador de microservicios (Aggregator Microservices)
+categoría: Arquitectónico
+idioma: es
+etiquetas:
+- Cloud distributed (Nube distribuida)
+- Decoupling (Desacoplamiento) 
+- Microservices (Microservicios)
+---
```
Comment (iluwatar) [inline localization/es/aggregator-microservices/README.md:9]: The yaml frontmatter keys are not supposed to be translated, only values.

**Review by iluwatar — CHANGES_REQUESTED** [review]

**Review by KarmaTashiCat — COMMENTED** [review]
I'm sorry , It's my first issue and pull request, and as mistake I've submitted before changes, but now I've made the changes hoping everything is ok according the recent commits from issue #2353 but I'm confused about the tag keyword because I was trying to figure out about YAML keys and reading some documentation saying that "tags" is the right keyword but now is switched for "tag" keyword. Sorry for my confusion, trying to figure it out the changes requested, I've forked and commited pullrequest before #2353 newest commits, I hope that now "tag" was expected in changes requested too instead old version "tags" keyword.
I will still paying attention to news about this question and if could it be merged or not yet, thank you for changes request @iluwatar I will try to considere it in future PRs.

File: localization/es/aggregator-microservices/README.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+title: Aggregator Microservices (Agregador de microservicios)
```
Comment (iluwatar) [inline localization/es/aggregator-microservices/README.md:2]:
```suggestion
title: Aggregator Microservices
```

File: localization/es/aggregator-microservices/README.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+category: Architectural (Arquitectónico)
```
Comment (iluwatar) [inline localization/es/aggregator-microservices/README.md:3]:
```suggestion
category: Architectural
```

File: localization/es/aggregator-microservices/README.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+language: es
+tag:
+- Cloud distributed (Nube distribuida)
```
Comment (iluwatar) [inline localization/es/aggregator-microservices/README.md:6]:
```suggestion
- Cloud distributed
```

File: localization/es/aggregator-microservices/README.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+- Decoupling (Desacoplamiento)
```
Comment (iluwatar) [inline localization/es/aggregator-microservices/README.md:7]:
```suggestion
- Decoupling
```

File: localization/es/aggregator-microservices/README.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
+---
+title: Aggregator Microservices (Agregador de microservicios)
+category: Architectural (Arquitectónico)
+language: es
+tag:
+- Cloud distributed (Nube distribuida)
+- Decoupling (Desacoplamiento) 
+- Microservices (Microservicios)
```
Comment (iluwatar) [inline localization/es/aggregator-microservices/README.md:8]:
```suggestion
- Microservices
```

**Review by iluwatar — CHANGES_REQUESTED** [review]
The current convention is to keep the yaml frontmatter untranslated, so let's stick to that. Otherwise, looks good.

**Comment by KarmaTashiCat** [comment]
> The current convention is to keep the yaml frontmatter untranslated, so let's stick to that. Otherwise, looks good.

Resolved! Ty for RC!

**Comment by iluwatar** [comment]
@KarmaTashiCat please rebase the branch on top of master. This should resolve the sonar analysis error.

**Review by iluwatar — APPROVED** [review]

**Comment by iluwatar** [comment]
Looks good! Thank you for the contribution 🎉

@all-contributors please add @KarmaTashiCat for translation

**Pull request merged**